Makefile

```diff
@@ -257,12 +257,22 @@ push_client_image:
 	@echo "To activate the venv27, execute 'source .venv27/bin/activate' or '.venv27/bin/activate.bat' (WIN)"
 
 travis-build:
```

Comment:
travis config sometimes uses direct calls (e.g. to ``pip install`` or ``pytest``) and sometimes relies on the makefile. Shouldn't there be some consistency. As a rule of thumb, everything in the travis config should be "runnable" from command line during development. 

I wonder if the solution is to have a makefile per service that is called from travis

Reply:
I think we should aim to have mostly everything in makefiles. This would make it easier to reproduce in command line in dev. I'll try to add more and more stuff in the makefiles with time